showcase/src/app/shared/components/example-viewer/example-viewer.component.ts

```diff
@@ -71,8 +72,8 @@ export class ExampleViewerComponent implements OnInit {
 	 */
 	public getAppBaseHref(): string {
 		// the final url in GitHub Pages will be something like "/showcase/latest/" or "/showcase/some-version/"
-		const finalUrlRegex: RegExp = /(\/showcase\/[\d\D][^\/]+(\/|\/$|$))/;
-		const trailingSlashRegex: RegExp = /\/$/;
+		const finalUrlRegex = /(\/showcase\/[\d\D][^/]+(\/|\/$|$))/;
```

Comment:
Why the character `\` before the `/` has been removed ?

Reply:
those character are not  necessary and break the rule. This is done automatically with the --fix option